src/request/remove-key/index.html

```diff
@@ -37,11 +37,14 @@
     <script defer bundle-toplevel src="../../translations/index.js"></script>
 
     <script defer src="../../lib/AutoComplete.js"></script>
+    <script defer src="../../lib/Iqons.js"></script>
+    <script defer src="../../lib/QrEncoder.js"></script>
```

Comment:
Needed here?

Reply:
Yes, as dependencies for the LoginFile component.

Reply:
(Unnecessary scripts would throw a `checkdeps` error during the Travis CI run.)